packages/badge/src/badge.css

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+@import './spectrum-badge.css';
+
+/* center align icons and text */
+
+:host {
+    display: inline-flex;
```

Comment:
This overrides https://github.com/adobe/spectrum-web-components/pull/2305/files#diff-5edf46dbb57c6fb3d202eb46df1dd1abd71d482a43702f41acc2478f3d765bfcR201 can you comment here why, if it's still needed.

Reply:
Yep, it's converting from `inline-block` to `inline-flex` in order to support the internal layout of `${icon} ${label}` (whereas the current upstream implementation only has labels, no icons).

For example, limiting the label to a max of 2 rows, allowing flexible shrinking & cropping of the label, while keeping the icon from shrinking, and maintaining a vertical center across both the icon and the label:

![image](https://user-images.githubusercontent.com/364501/170528350-b84881da-b3c8-44f1-b3e4-6b59f62b7638.png)